prow/config/config.go

```diff
@@ -96,6 +99,23 @@ type Plank struct {
 	ReportTemplate *template.Template `json:"-"`
 }
 
+// JenkinsOperator is config for the jenkins-operator controller.
+type JenkinsOperator struct {
```

Comment:
Do we expect these two to drift? It seems like we would have a reasonable interface for config between the two of them

Reply:
The jenkins-operator already has its own flags that potentially can move inside the config. So yes, I expect drift. But even if they never drift, we cannot use a global config for both. We already have issues with our kube jobs not showing up correctly in deck because of the job template and we need to set max_concurrency per agent.  We definitely need a common interface between controllers though but out of scope for this PR.